tests/testthat/test-utilities.R

```diff
@@ -8,3 +8,21 @@ test_that("It returns the correct name of an enum entry", {
 test_that("It returns NULL if the entry does not exist", {
   expect_equal(netEnumName(enumType = enumType, 5L), NULL)
 })
+
+context("createConfigurationForFile")
```

Comment:
I think the tst should be moved to `test-data-importer-configuration.R` where all other data importer configurtion tests are stored.